test/spec_helper.rb

```diff
@@ -0,0 +1,10 @@
+require "yaml"
```

Comment:
see helpers in other yast modules. Important part is set Y2DIR, which then ensure that require yast set load path to your working directory - https://github.com/yast/yast-bootloader/blob/master/test/test_helper.rb#L1

We also have rspec helpers in yast, which can help you